staging/src/k8s.io/kubectl/pkg/cmd/expose/expose_test.go

```diff
@@ -611,6 +611,8 @@ func TestRunExposeService(t *testing.T) {
 					switch p, m := req.URL.Path, req.Method; {
 					case p == test.calls[m] && m == "GET":
 						return &http.Response{StatusCode: test.status, Header: cmdtesting.DefaultHeader(), Body: cmdtesting.ObjBody(codec, test.input)}, nil
+					case p == test.calls[m] && m == "POST":
```

Comment:
Why do you have to change that?

Reply:
As mentioned, the test actually was broken but somehow was passing... The table test's `test.output` was never used until now, and we somehow got away with returning the `test.input` as part of the test response, which works for part of the expose command.